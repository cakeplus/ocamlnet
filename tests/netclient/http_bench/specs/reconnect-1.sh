. helpers.sh

# The -sleep is necessary to ensure that both GET /second and GET /third
# are sent to the server. Otherwise, the results are non-deterministic.

start_test_server \
    -line 1 -file data/framed-8bitclean \
    -sleep 1 \
    -line 6 -file data/illegal-protocol -reconnect \
    -line 1 -file data/chunked \
    -line 6 -file data/framed-8bitclean
trap "stop_test_server" EXIT
request -get /first -get /second -get /third -run
