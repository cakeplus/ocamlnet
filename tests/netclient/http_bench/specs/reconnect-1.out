FB00E283C6995B96668F4876CE199F28
1E533DFF0CEFF29D994D083FF960FAA2
FB00E283C6995B96668F4876CE199F28
